plugins/ca-data-standards/plugin.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ca-data-standards",
+  "version": "0.0.1",
+  "description": "Plugin for validating Avro schemas against data standards locally",
+  "packages": [
+    "nodejs@20.18.0",
+    "pnpm@9.12.3"
```

Comment:
```suggestion
```
Having corepack enabled will provide this plugin with pnpm. If we include pnpm as a package here we can have an issue with the node version pnpm brings being different to the node version in the devbox package.